src/main/scala/com/mozilla/telemetry/utils/package.scala

```diff
@@ -179,4 +179,13 @@ package object utils{
     val path = new Path(pathString)
     FileSystem.get(path.toUri, new Configuration()).exists(path)
   }
+
+  /** Detect existence of a directory or file using hadoop
```

Comment:
Docstring doesn't match method signature, did you mean:
```suggestion
  /** Delete file or directory in Hadoop filesystems
```